src/commands/scaling/addScaleRule/IAddScaleRuleWizardContext.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { ScaleRuleGroupTreeItem } from "../../../tree/ScaleRuleGroupTreeItem";
+
+export interface IAddScaleRuleWizardContext extends IActionContext {
+    treeItem: ScaleRuleGroupTreeItem;
+    ruleName?: string;
+    ruleType?: string;
+    concurrentRequests?: string;
+    queueName?: string;
+    queueLength?: string;
```

Comment:
It's a matter of preference, but I usually define the context values to match the "true" value is going to be at the time of the API call. What I mean is, queueLength will come in as a string because of user input, but when being sent in the request, it should be a number.

Therefore, I would do the conversion in the prompt step rather than at the time of the API call.  The benefit being, if we ever use that step again for something else, we wouldn't have to remember to convert it to a number.